README.md

```diff
@@ -76,24 +76,40 @@ instruct Vagrant to load it.
 #### Use Libvirt as provider
 
 In order to use libvirt as provider you need to set
-the value of `vagrant_provider` variable to `libvirt` inside `env.yaml`.
+the value of `conf.vagrant_provider` variable to `libvirt` inside `env.yaml`.
 Vagrant needs to know that you want to use libvirt and not default VirtualBox,
 then you can use the option `--provider=libvirt`.
 
+#### Select a kubernetes networking plugin (CNI)
+
+The networking plugin is configured by setting `ansible.group_vars.all.kubernetes_network_plugin`
+inside `env.yaml`. The currenlty allowed plugins are `weavenet`, `calico` and `flannel`. It is
+also possible to use `no-cni-plugin`. In this case, only the basic configuration of kubernetes will
```

Comment:
Let's avoid the passive form (less readable).

In this case, the provisioner will only prepare the environment, but will not start any Kubernetes cluster

Reply:
fixed, thanks